internal/ark/sync/backups_sync_svc.go

```diff
@@ -117,6 +117,7 @@ func (s *BackupsSyncService) SyncBackupsForCluster(cluster api.Cluster) error {
 	if err != nil {
 		return errors.WrapIf(err, "could not list backups")
 	}
+	backupIDS := make([]int, 0)
```

Comment:
**Recommended**: could be `backupIDs := make([]int, 0, len(backups.Items))` to optimize allocations in general case (and less than half of the items running into error cases) while also keeping the `append()` style filling.

Reply:
fixed